### PR TITLE
feature/improve-readability

### PR DESCRIPTION
**Pull request by KennethanCeyer**
## 💬 Comment

- This is a draft of the Post page design. In other words, the shape of this PR is not the final draft.
- Code snippets will be highlighted in a future PR.

⚠ **IMPORTANT**
-  I am reviewing whether each element of Gatsby can be changed to a React Component. If you have any ideas, please comment.

## 🚀 Features

- Improve post page design
- Add more example description of `zap.md`
- Add new plugin for captioning the image
- Fix all the typography

image
image

### CODE REVIEW

**Review by zi-gae — DISMISSED** [review]
I think I like the this PR.

Thanks!

File: src/components/posts/PostContent.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { dimensions } from 'styles/variables';
+
+interface ArticleHeaderProps {
+    dangerouslySetInnerHTML: {
+        __html: string;
+    } | undefined;
+}
```
Comment (zi-gae) [inline src/components/posts/PostContent.tsx]: I suggestion a proposal. **optional** is simple.
```suggestion
interface ArticleHeaderProps {
    dangerouslySetInnerHTML?: {
        __html: string;
    };
}
```

Comment (KennethanCeyer) [reply src/components/posts/PostContent.tsx]: Think it is better style
Thank you 👍

Comment (KennethanCeyer) [reply src/components/posts/PostContent.tsx]: Switch to resolved

**Pull request merged**